packages/admin/src/components/Navigation/StyledComponents.js

```diff
@@ -1,53 +1,52 @@
 import styled from 'styled-components'
 import {Link} from 'react-router-dom'
-
-export const StyledSearch = styled.input`
-  border-radius: 20px;
-  border: 0px;
-  width: 90%;
-  font-size: 1.2rem;
-  padding: 4px 7px;
-  :focus {
-      outline: none;
-  }
-  margin-bottom: 5px;
-`
+import {Button, theme} from 'tocco-ui'
 
 export const StyledNav = styled.nav`
   height: 100%;
 `
 
-export const StyledMewnuWrapper = styled.div`
+export const StyledMenuWrapper = styled.div`
   height: 100%;
   overflow: auto;
+  padding-left: 2rem;
+  
+  div > div:last-child {
+    padding-bottom: 2rem;
+  }
+
   ::-webkit-scrollbar {
-      width: 5px;
+      width: 10px;
   }
    
   ::-webkit-scrollbar-thumb {
-    background-color: #878787;
+    background-color: ${theme.color('text')};
+    
+    &:hover {
+      background-color: ${theme.color('primaryBlue')};
+    }
   }
   
-  scrollbar-color: #878787 #253653; // Firefox workaround
+  scrollbar-color: ${theme.color('text')} transparent; // Firefox workaround
   scrollbar-width: thin;
 `
 
 export const StyledMenuEntry = styled.span`
-   color: #c5cbd4;
-   font-weight: bold;
+   color: ${theme.color('text')};
+   font-weight: ${theme.fontWeight('bold')};
 `
 
 export const StyledMenuLink = styled(Link)`
-  color: #c5cbd4;
+  color: ${theme.color('text')};
   text-decoration: none;
+  padding-left: 1.3rem;
   
   &:hover {
-      color: #555;
+      color: ${theme.color('secondaryBlue')};
     }
   
   &:focus {
-     text-decoration: underline;
-      outline: none;
+     outline: none;
```

Comment:
i think text-decoration or something similar is still needed. It is possible to select a link with tab. But witout a distinction in styling there is no way to tell what link has the focus.